README.md

```diff
@@ -267,6 +267,15 @@ export const enum PermissionKey {
   DeleteAudit = 'DeleteAudit',
 }
 ```
+- Serving the static files:
+
+Authorization  configuration binding sets up paths that can be accessed without any authorization checks, allowing static files to be served directly.The allowAlwaysPaths property is used to define these paths for the files in public directory  i.e for a test.html file in public directory ,one can provide its path as follows:
+
+```
+this.bind(AuthorizationBindings.CONFIG).to({
+  allowAlwaysPaths: ['/explorer',''/test.html'],
+});
+```
```

Comment:
also mention that the public folder should be added to static 
and if the file is in some other folder then.. one more app.static has to be added.

Reply:
ok

Reply:
Done.